settings-migration/upload-settings.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+if [ -z "$2" ]; then
+  echo "Usage: $0 <namespace> <folder with settings to upload>"
+  exit 1
+fi
+
+NAMESPACE=$1
+OUTPUT_DIR=$2
+
+kubecost_aggregator=$(kubectl get pod -n $NAMESPACE -l app=aggregator -o jsonpath='{.items[0].metadata.name}')
+
+
+if [ -f $OUTPUT_DIR/advanced-reports.json ]; then
+  kubectl cp -n $NAMESPACE -c aggregator $OUTPUT_DIR/advanced-reports.json ${kubecost_aggregator}:/var/configs/advanced-reports.json
+  echo "Uploaded advanced-reports.json"
+fi
+if [ -f $OUTPUT_DIR/apiconfig.json ]; then
+  kubectl cp -n $NAMESPACE -c aggregator $OUTPUT_DIR/apiconfig.json ${kubecost_aggregator}:/var/configs/apiconfig.json
+  echo "Uploaded apiconfig.json"
+fi
+if [ -f $OUTPUT_DIR/asset-reports.json ]; then
+  kubectl cp -n $NAMESPACE -c aggregator $OUTPUT_DIR/asset-reports.json ${kubecost_aggregator}:/var/configs/asset-reports.json
+  echo "Uploaded asset-reports.json"
+fi
+if [ -f $OUTPUT_DIR/budgets.json ]; then
+  kubectl cp -n $NAMESPACE -c aggregator $OUTPUT_DIR/budgets.json ${kubecost_aggregator}:/var/configs/budgets.json
+  echo "Uploaded budgets.json"
+fi
+if [ -f $OUTPUT_DIR/cloud-configurations.json ]; then
```

Comment:
Does this live on the aggregator pod? I only see it on the cost-analyzer pod in kc-demo.